picire/cli.py

```diff
@@ -108,7 +108,7 @@ def process_args(parser, args):
     else:
         args.encoding = chardet.detect(args.src)['encoding']
         if not args.encoding:
-            parser.error('The encoding of the test is not recognized.'
+            parser.error('The encoding of the input file is not recognized. '
```

Comment:
For the sake of consistency, I'd rather rename `test` to `test case` here and similarly, at the previous chunk, `Input file` to `Test case`. (The help strings of the CLI options and the other comments refer to the input as `test case`.)

Reply:
Sure thing, done!

Reply:
Thanks! Landed.